pyproject.toml

```diff
@@ -35,7 +35,16 @@ numpy = "^1.19.4"
 pytest = "^6.2.1"
 pytest-aiohttp = "^0.3.0"
 pytest-cov = "^2.11.0"
+black = "^22.1.0"
 
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.black]
+line-length = 120
+
+[tool.isort]
```

Comment:
I think we should use pyproject or setup.cfg for this configuration. isort has configuration on both files and this can cause unexpected behaviors. Maybe configure all (including black) on setup.cfg or bring the configurations there to pyproject

Reply:
You are right, I didn't notice we had setup.cfg already